browsers-adapters/chrome/worker-listeners.js

```diff
@@ -1,15 +1,6 @@
 import { handleTabVisit } from "../../background.js";
 import * as actions from "./worker-actions.js";
 
-// Listen when user switches to a tab (activates it)
-chrome.tabs.onActivated.addListener((activeInfo) => {
```

Comment:
Removed as it was causing the warning to be triggered everytime I switch tabs. Plugin works without it. Was there a specific case this was handling @IbrahimMohammed47 ?

Reply:
Yes, it was just for that same reason: to be triggered everytime I switch tabs. XD
Which is not that annoying anyway